pkgs/applications/networking/irc/weechat/scripts/colorize_nicks/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, lib, fetchurl }:
+
+stdenv.mkDerivation {
+  pname = "weechat-colorize_nicks";
+  version = "27";
+
+  src = fetchurl {
+    url = "https://github.com/weechat/scripts/raw/bc8a9051800779a036ba11689a277cd5f03657b2/python/colorize_nicks.py";
```

Comment:
This won't auto update which is fine here I guess?

Reply:
No, it cannot. Since the repo tracks multiple scripts there is also no usable tag. This is fine.